ui/page/components/wallet_sync_info.go

```diff
@@ -322,10 +322,37 @@ func (wsi *WalletSyncInfo) syncContent(gtx C, uniform layout.Inset) D {
 		syncIsScanning = wsi.wallet.(*dcr.Asset).IsSycnRescanning()
 	}
 	dp8 := values.MarginPadding8
+
+	totalStep := 2
+	if !wsi.isBtcOrLtcAsset() {
+		totalStep = 3
+	}
+	syncStep := 1
+	syncProgress1 := values.String(values.StrFetchingBlockHeaders)
+	if wsi.wallet.IsSyncing() {
+		if !wsi.isBtcOrLtcAsset() {
+			if isAddDiscovering {
+				syncStep = 2
+				syncProgress1 = values.String(values.StrAddressDiscovering)
```

Comment:
isn't address discovery also a thing for btc and ltc?

why is it only applying for dcr

Reply:
this is old logic from dmigwi's code, I have checked it, btc and ltc not address discovery